user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+
+Manage Your User Details
+^^^^^^^^^^^^^^^^^^^^^^^^
+|
+
+.. image:: ./img/intro/user_edit_general.png
+
+|
+
+At the top of the *"General"* section, you can view, define and change the basic details such as:
+
+- **Username**: An alias used for login and identification. It should be at least three characters long and cannot 
+  start with a space.
+
+- **Names**: Your personal name. It can be comprised of the first, middle, and last name, a prefix and a suffix. The 
+  first and the last name are mandatory. You can use any number of characters of any alphabet in the name.
+
+- **Birthday**: Select the date in the calendar. 
+
+- **Avatar**: A picture associated with you in the system. To set an avatar, click the :guilabel:`Choose file` button 
+  and select your avatar file from your computer. Avoid using large files; 100×100 pixels is recommended. After you have 
+  saved the user settings the file will be checked for consistency with your organization’s rules.  Please contact your 
+  OroCRM administrator for details. The image will be displayed along with your name in the selectors and will help 
+  other users to recognize you quickly.
+
+  |
+  
+  |EditAvatar|
+
+  |
+  
+Any custom details permitted for editing will be available in the *"Additional"* section. In the example above, this is 
+the *"Title"* field.
+
+.. note::
+
+    Although the *"Status"* fields is displayed on the Edit user page, you cannot change it, as it is impossible to 
+    deactivate a profile that you are using.
+
+    
+Manage Your Contact Details
+^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+ 
+You can define and update the contact information in  the *"General"* section. 
+
+|
+
+.. image:: ./img/intro/user_edit_contacts.png 
+
+|
+  
+- **Your email addresses**: You are required to provide a primary email address. This email address must be unique for 
+  each user in the system. You can also add additional email addresses via :guilabel:`Add Another Email` button. 
+  
+  These addresses are used when an email is created within OroCRM: 
+
+ - The primary email address is used as a default value and additional addresses are available in the selector for the 
+   *"from"* field.
+
+   |
+   
+   |EmailFrom|
+
+   |
+
+.. note::
+
+     If synchronization with an external SMTP server has been enabled for an external mailbox (as described in the 
+     :ref:`Synchronize Your Mailbox with OroCRM section <user-guide-intro-log-in-edit-profile-synchronize>` below, 
+     your addressee will always see the address of the synchronized mailbox as the *"from"* field value.
+
+ - You will see the letters sent by other OroCRM users to your primary and additional email addresses in the 
+   "Activities" section on your User page.
+   
+   |
+   
+   |EmailTo|   
+   
+   |
+   
+- **Phone number**: is displayed to other users among your contact details and will be used as a default value to log a 
+  call related to you or call you via integrated phone applications, such as 
```

Comment:
Edit: **Phone number**: This is displayed to the other users among your contact details, and will be used as a default value to log a call related to you or to call you via integrated phone applications, such as